aptos-move/framework/aptos-framework/sources/object.move

```diff
@@ -49,6 +49,16 @@ module aptos_framework::object {
     /// nesting, but any checks such as transfer will only be evaluated this deep.
     const MAXIMUM_OBJECT_NESTING: u8 = 8;
 
+    /// Scheme identifier used to generate an object's address `obj_addr` as derived from another object.
+    /// The object's address is generated as:
+    /// ```
+    ///     obj_addr = sha3_256(account addr | derived from object's address | 0xFC)
+    /// ```
+    ///
+    /// This 0xFC constant serves as a domain separation tag to prevent existing authentication key and resource account
+    /// derivation to produce an object address.
+    const OBJECT_DERIVED_SCHEME: u8 = 0xFC;
```

Comment:
really need a centralized place for those domain separators.

Reply:
Where else would they be outside of object?

Reply:
resource acct scheme

Reply:
Good catch! Let's use a different value here then

Reply:
Since constants are not public currently. We'll need to provide getter functions which is kinda annoying though